.eslintrc

```diff
@@ -1,10 +1,19 @@
 {
   "extends": "@folio/eslint-config-stripes",
   "parser": "babel-eslint",
+  "plugins": ["react-hooks"],
   "rules": {
     "max-len": ["warn", { "code": 120 }],
     "operator-linebreak": ["off"],
-    "react/sort-prop-types": ["error"]
+    "react/sort-prop-types": ["error"],
```

Comment:
I'd rather we just agree on what rules we want and set them in `eslint-config-stripes`